c2corg_ui/templates/i18n.html

```diff
@@ -118,6 +118,9 @@
 <span translate>You have no rights to edit this document.</span>
 <span translate>Form is not valid</span>
 <span translate>You must log in to edit this document.</span>
+<span translate>no associated routes</span>
```

Comment:
It's probably better to use a capital initial letter for those messages (in the translations as well).